src/features/copyVariableName.ts

```diff
@@ -7,7 +7,7 @@ export const registerCopyVariableName = () => {
         if (!activeEditor || activeEditor.viewColumn === undefined) return
         // also would make sense to support mutliple selections/cursors
         const lineText = activeEditor.document.lineAt(activeEditor.selection.end).text
-        const regexps = [/\s*(?:const(?: {)?|let(?: {)?|type|interface|import(?: {)?) ([\w\d]+)/, /\.([\w\d-]+?)\s*{/]
+        const regexps = [/\s*(?:const(?: {)?|let(?: {)?|type|interface|import(?: {)?) ([\w\d]+)/, /\s*(?:\.|(?:.*(?:class|className)=["']))([\w\d-]+?)\s*[{"']/]
```

Comment:
```ts
/\s*(?:\.|(?:.*(?:class|className)=["']))([\w\d-]+?)\s*[{"']/
//                                                      ^ whats the purpose of this char?
```

Reply:
I just bought 18 nuggets wishing to relive every moment with you

Reply:
And two cheese toppings

Reply:
@sa2urami  sa2uram

Reply:
@sa2urami zardo#21428 battle

Reply:
I'm afraid typescipt is malignant tumour which disable ability to check tg

Reply:
@sa2urami  I can say the same about ur ds